<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreadIdFixer</class>
 <widget class="QMainWindow" name="ThreadIdFixer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ThreadIdFixer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0" colspan="2">
     <widget class="QTextEdit" name="teLog"/>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Processed</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Errors</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="leProcessed"/>
      </item>
      <item row="0" column="3" rowspan="2">
       <widget class="QPushButton" name="pbGo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Go</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="leErrors"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leFileName">
        <property name="text">
         <string>post.log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbSelect">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
